README: document how to run phase1 (#141)
ZcashFoundation/frost-zcash-demo: README.md

example below.

## Socket communication example

Create 3 key shares with threshold 2 using trusted dealer:

```
cargo run --bin trusted-dealer -- -t 2 -n 3
```

The key packages will be written to files. Securely send the partipant's key
packages to them (or just proceed if you are running everything locally for
testing).

Start a signing run as the coordinator:

```
cargo run --bin coordinator -- -i 0.0.0.0 -p 2744 -n 2 -m message.raw -s sig.raw
```

This will start a server listening for connections to any IP using port 2744.
(These are the default values so feel free to omit them.) The protocol will run
with 2 participants, signing the message inside `message.raw` (replace as
appropriate). The signature will be written to `sig.raw`. The program will keep
running while it waits for the participants to connect to it.

Each participant should then run (or run in different terminals if you're
testing locally):

```
cargo run --bin participant -- -i 127.0.0.1 -p 2744 -k key-package-1.json
```

It will connect to the Coordinator using the given IP and port (replace as
needed), using the specified key package (again replace as needed).

> When two participants run, the Coordinator should complete the protocol and
> write the signature to specified file.

Reviewer: ```suggestion
Once two participants are running, the Coordinator should complete the protocol and
write the signature to specified file.
```

Reviewer: I think this makes it a bit clearer
